Emby.Server.Implementations/Library/UserManager.cs

```diff
@@ -400,7 +407,10 @@ private IAuthenticationProvider[] GetAuthenticationProviders(User user)
 
             if (providers.Length == 0)
             {
-                providers = new IAuthenticationProvider[] { _defaultAuthenticationProvider };
+                // this function used to assign any user without an auth provider to the default.
+                // we're going to have it use a new function now.
+                _logger.LogWarning($"The user {user.Name} was found but no Authentication Provider with ID: {user.Policy.AuthenticationProviderId} was found. Assigning user to InvalidAuthProvider temporarily");
```

Comment:
Let the logger handle the interpolation

Reply:
I redid the comment and log message using proper formatting and including a slightly nicer (IMO) wording.